### DISCUSSION TITLE
UEFI flag "OS Chain A status" change to unbootable after adding a file in /etc

### DISCUSSION THREAD

**Discussion by MichaelPres** [OE4T/meta-tegra#1528] [Q&A]
I did a simple test by creating a new random file in /etc/test123.cfg and then reboot. After a few time, the Orin stop booting. After investigating, I found that "OS Chain A status = Unbootable" in UEFI bootloader. Putting this flag back to "Normal" make it bootable again.

Do you have any clue on what is causing this?

I am using core-image-minimal-jetson-agx-orin-devkit.tegraflash.tar.gz from tegra-demo-distro branch kirkstone on an Orin Agx.

UEFI print screen: ![image](https://github.com/OE4T/meta-tegra/assets/147830576/50d9aef5-13da-4a34-9af8-862ff2eed582)
TeraTerm log: [tegra-demo-orin_stop_booting.txt](https://github.com/OE4T/meta-tegra/files/14951070/tegra-demo-orin_stop_booting.txt)

**Comment by dwalkes** [comment]
My guess is it's not related to the `/etc/test123.cfg` file but rather whatever service (`nvstartup`?) nvidia uses to mark successful boot is not getting included in core-image-minimal and therefore the slot is getting marked unbootable by UEFI after 3 unsuccessful boots, similar to the discussion in https://forums.developer.nvidia.com/t/l4tlauncher-would-like-to-avoid-being-bricked-when-both-a-b-slots-are-marked-as-unbootable/288078

If you are just looking for something to get started with I'd recommend demo-image-base at https://github.com/OE4T/tegra-demo-distro

If you have a specific need to use core-image-minimal I'd look at the startup services included there and how they differ from the demo-image-base image.

**Comment by MichaelPres** [comment]
I think it is :
```
    setup-nv-boot-control 
    setup-nv-boot-control-service
```
By the way, why packagegroup in meta-tegrademo are not in meta-tegra?
